src/core/thread/mesh_forwarder.cpp

```diff
@@ -2271,4 +2279,75 @@ void MeshForwarder::HandleDataPollTimeout(void)
     }
 }
 
+#if OPENTHREAD_CONFIG_LOG_LEVEL >= OPENTHREAD_LOG_LEVEL_INFO
```

Comment:
`Info` makes sense for this. It's not a warning or error and we want this at a level higher than debug. What would you suggest for it to be?